x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -134,6 +136,46 @@ const CustomGridBody = memo(
   }
 );
 
+/**
+ * An error callout that displays the error stack in a code block
+ */
+const ViewError = ({ error }: { error: Error }) => (
+  <>
+    <EuiFlexGroup
+      gutterSize="s"
+      alignItems="center"
+      // Here we force the error callout to be the same height as the cell content
+      // so that the error detail gets hidden in the overflow area and only shown in
+      // the cell popover
+      css={css`
```

Comment:
tiny nitpick but for static props like styles I think it's better if we assign them to static variables and put them outside of the components (if they don't depend on internal component state).